README.md

```diff
@@ -23,15 +23,15 @@
   </a>
 </p>
 
-Turbo is an next-generation toolchain for frontend development, written in Rust. It consists of 3 major parts:
+Turbo is a next-generation, Blazing Fast™, toolchain for frontend development, written in Rust. It consists of 3 major parts:
 
 - [**Turbopack:**](https://turbo.build/pack) an incremental bundler (the successor to Webpack)
 - [**Turborepo:**](https://turbo.build/repo) an incremental build system
-- The Turbo engine: a low-level incremental computation and memoization engine
+- The Turbo engine: a Blazing Fast™ low-level incremental computation and memoization engine
 
 ## Getting Started
 
-Visit https://turbo.build to get started with Turbopack and Turborepo.
+Visit https://turbo.build to get started with Turbopack and Turborepo (Hand crafted by Vercel which is why its Blazing Fast™).
```

Comment:
Can we rewrite this part in rust?

Reply:
@ryanwinchester Yes.

Reply:
This needs more fish to go faster...
https://turbo.fish/